src/FixedDataTable.js

```diff
@@ -437,7 +437,7 @@ class FixedDataTable extends React.Component {
   }
 
   componentWillMount() {
-    this._didScrollStop = debounceCore(this._didScrollStopSync, 200, this);
+    this._didScrollStop = debounceCore(this._didScrollStopSync, 50, this);
```

Comment:
Should we move the delay to a configurable prop, or for now to a constant

Reply:
moved to constant